__tests__/app.test.js

```diff
@@ -36,17 +44,82 @@ describe("GET/api/articles/:article_id", () => {
     return request(app)
       .get("/api/articles/1")
       .expect(200)
-      .then(({ body }) => {
-        expect.objectContaining(body);
+      .then(({ body: article }) => {
+        expect(article).toHaveLength(1);
+        article.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number)
+            })
+          );
+        });
       });
   });
-  test("404:article id not in database", () => {
+  test("400:article id not in database", () => {
```

Comment:
Article id not in database should be a 404 not found! A 400 should be used for an article id that's of the wrong type.